pkg/selinux/setroubleshoot-client.js

```diff
@@ -35,9 +35,10 @@ var busNameFixit = "org.fedoraproject.SetroubleshootFixit";
 var dbusInterfaceFixit = busNameFixit;
 var dbusPathFixit = "/org/fedoraproject/SetroubleshootFixit/object";
 
-client.init = function() {
+client.init = function(capabilitiesChangedCallback) {
```

Comment:
This capabilitiesChanged concept is weird:

1. It only ever changes once (a simple supports() which returns a promise is enough)
2. Using D-Bus introspection data for capabilities checking is awkward and makes the code much more complicated than it needs to be. This is only for the transition right? Can't we just make cockpit depend on the new version of setroubleshoot-server?